src/plugins/hexnumber/README.md

```diff
@@ -0,0 +1,31 @@
+- infos = Information about the template plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = nodep experimental unfinished
+- infos/metadata =
+- infos/description = interprets strings with prefix 0x as hexadecimal numbers
```

Comment:
`infos/ordering = type` is needed: you want to be sure that your plugin runs before the type plugin, otherwise the type plugin would reject numbers with `0x..`.